html-report/src/components/LogFilter.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import SearchBar from './SearchBar';
+import LogFilterButton from './LogFilterButton';
+
+const SEARCH_FIELDS = ['level', 'line'];
+const SEARCH_REF = 'line';
+export default class LogFilter extends Component {
+  static propTypes = {
+    setSearchResults: PropTypes.func,
+    data: PropTypes.array
+  };
+
+  render() {
+    return (
+      <div>
+        <div className="card">
+          <div className="title-common">LOGS</div>
+          <LogFilterButton onClick={ () => this.performFilterSearch("level:verbose") } text="Verbose" disabled={ !!!this.props.data.length } />
```

Comment:
Using `!!!` is not a good practice — what you basically do is converting a number to a boolean with the `!!` and checking the opposite value. In JS you can use just number to check for `true` or `false`. If the number is `0` — it's a false value, if ` > 0` — it's a true value.